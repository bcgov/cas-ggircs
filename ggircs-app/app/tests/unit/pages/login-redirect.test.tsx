import { shallow } from "enzyme";
import LoginRedirect from "pages/login-redirect";

describe("re-direct", () => {
  it("matches the last accepted Snapshot", () => {
    const wrapper = shallow(
      <LoginRedirect
        query={{
          session: null,
        }}
        router={{
          query: {
            sessionIdled: false,
          },
        }}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});

describe("re-direct session idled", () => {
  it("matches the last accepted Snapshot", () => {
    const wrapper = shallow(
      <LoginRedirect
        query={{
          session: null,
        }}
        router={{
          query: {
            sessionIdled: true,
          },
        }}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
