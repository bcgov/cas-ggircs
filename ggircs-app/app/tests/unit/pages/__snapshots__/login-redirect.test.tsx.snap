// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`re-direct matches the last accepted Snapshot 1`] = `
<Relay(DefaultLayout)
  session={null}
>
  <Row
    noGutters={false}
  >
    <Col
      md={6}
    >
      <h3
        className="blue"
      >
        You need to be logged in to access this page.
      </h3>
      <p>
        Please log in to access this page.
        <br />
        You will be redirected to the requested page after doing so.
      </p>
    </Col>
  </Row>
</Relay(DefaultLayout)>
`;

exports[`re-direct session idled matches the last accepted Snapshot 1`] = `
<Relay(DefaultLayout)
  session={null}
>
  <Row
    noGutters={false}
  >
    <Col
      md={6}
    >
      <h3
        className="blue"
      >
        You were logged out due to inactivity.
      </h3>
      <p>
        Please log in to access this page.
        <br />
        You will be redirected to the requested page after doing so.
      </p>
    </Col>
  </Row>
</Relay(DefaultLayout)>
`;
